index.html

```diff
@@ -76,9 +83,9 @@ <h1 class="project-name">Tonic</h1>
                         <li class="css"><a class="cl" href="#">css</a></li>
                         <li class="javascript"><a class="cl" href="#">javascript</a></li>
                     </ul>
-                    <div class="project-link">
+                    <button class="project-link">
                         <a href="#" class="see-project">See Project</a>
-                    </div>   
+                    </button>   
```

Comment:
- [ ] I see you have implemented the button interactions, good job, let's make sure they are also following the figma template currently your buttons on hover is like this 
![image](https://user-images.githubusercontent.com/97521557/208231651-52b7f8f8-7b5c-4315-aa66-969a7e349f61.png)

How it is supposed to be from the figma template see 
![image](https://user-images.githubusercontent.com/97521557/208231678-397d2c7f-b7df-4d7d-acf2-0b2b05172798.png)

Please keep a consistent design and implement this for all your buttons

Reply:
I changed the hover functionality, thank you for the recommendation.